{"name":"Local: Exception Handling","url":"d:\\Python\\Exception Handling.py","tests":[{"id":1756999915815,"input":"","output":""}],"interactive":false,"memoryLimit":1024,"timeLimit":3000,"srcPath":"d:\\Python\\Exception Handling.py","group":"local","local":true}